doc/onboarding.md

```diff
@@ -35,51 +36,99 @@ Boulder Experimental Mathematics Lab.)
     https://code.visualstudio.com/docs/editor/extension-marketplace.
 
 2. If you don't have [Git](https://git-scm.com/) installed, install it. We use
-   Git to keep track of the different versions of our code.
+   Git to keep track of the different versions of our code. _Mac_: The easiest
+   way to install git is to install the Mac "developer tools." You can do so
+   by opening Settings > General > Software Update and then (in a terminal, as
+   mentioned above) executing `xcode-select --install`. Then follow the
+   prompts in the Software Update window; your computer may need to restart
+   for the changes to take effect.
 3. If you don't have [NodeJS](https://nodejs.org/en/) installed, install it.
    NodeJS allows you to run JavaScript outside of a web browser. We use it in
-   our front end code base.
+   our front end code base. _Mac_: You can obtain Node by first installing the
+   "node version manager" via:
+
+    ```
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.2/install.sh | bash
+    ```
+
+    [Note that you may need to change the `v0.40.2` portion of the above URL
+    to the latest version of nvm; it was current at the time of this writing,
+    but you can always find the latest release on its
+    [GitHub page](https://github.com/nvm-sh/nvm/releases).] Now either log out
+    and log back in for the nvm installtion to take effect, or instead just
+    execute `. ~/.nvm/nvm.sh`. Finally to install Node, choose the version of
+    Node you want (latest was 23 at the time of this writing, but you can
```

Comment:
this parens never got closed

Reply:
I feel like the period should follow the close parens, not the other way around. (sorry for being nitpicky)

Reply:
No, let's get it right. How's this?

Reply:
well now there's an extra space after the link :)

Reply:
Oh. You don't like how that looks? Would you like me to take out the space or will you? Either way, just let me know.

Reply:
Hopefully the nth time is the charm!